src/client/dfuse/dfuse_main.c

```diff
@@ -99,6 +99,21 @@ dfuse_launch_fuse(struct dfuse_info *dfuse_info,
 	return false;
 }
 
+static void
+show_help(char *name)
+{
+	printf("usage: %s -m <mountpoint>\n"
```

Comment:
Do you mean by this that all others options -s/-p/-c/-g/-S/-f are not required ?
Also it may be interesting to detail that without -p and -c, all system's/daos_server's pools+containers will be available, with -p only all containers in this pool, with -p and -c only this single container.

Reply:
-m and -s are required, all others are optional.  I've updated the text for this.

I'll be adding more detail of what the options mean in another PR.